formstack-consents/src/main/scala/com/gu/identity/formstackconsents/IdentityClient.scala

```diff
@@ -0,0 +1,49 @@
+package com.gu.identity.formstackconsents
+
+import com.gu.identity.formstackconsents.FormstackClient.FormstackConsent
+import com.gu.identity.globalConfig.DevConfig
+import io.circe.syntax._
+import cats.syntax.either._
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.generic.extras._
+import scalaj.http.Http
+
+class IdentityClient(config: DevConfig) extends StrictLogging {
+  def sendConsentToIdentity(formstackConsent: FormstackConsent, newsletter: Newsletter): Either[Throwable, Unit] = {
+
+    // The key in the JSON sent to Identity depends on the listType. Sometimes the listType is 'set-lists' with a value of the consent name,
+    // and sometimes the listType is 'set-consents'. See example below.
+    //  {
+    //    "email" : "lauren.emms@guardian.co.uk",
+    //    "set-consents" : "holidays"
+    //  }
+
+    implicit val circeConfig: Configuration = Configuration.default.copy(
+      transformMemberNames = {
+        case "listType" => newsletter.listType
+        case other => other
+      }
+    )
+
+    @ConfiguredJsonCodec case class IdentityRequest(email: String, listType: List[String])
```

Comment:
The reason I have defined this case class (and above config) within the class and not in a companion object is because a key of the class (listType) changes depending on the newsletter (see comment starting on line 14)